src/gadgets/CensorshipResistanceGadget.sol

```diff
@@ -0,0 +1,84 @@
+pragma solidity ^0.8.19;
+
+import {AttestationGadget} from "src/gadgets/AttestationGadget.sol";
+
+import {IAndromeda} from "src/IAndromeda.sol";
+
+abstract contract CensorshipResistanceGadget is AttestationGadget {
+    function Suave() internal virtual returns (IAndromeda);
```

Comment:
What's the purpose of this function?

To force the contract using the CRgadget to have IAndromeda instance as part of it?